test/abort.lua

```diff
@@ -7,7 +7,15 @@ local clock = require('clock')
 local watchdog = require('watchdog')
 
 local t0 = clock.monotonic()
-watchdog.start(1)
+
+local enable_coredump = arg[1]
+if enable_coredump == 'true' then
+    enable_coredump = true
+else
+    enable_coredump = false
```

Comment:
I'd suggest eliminating the `else` branch. It would allow testing the default value.

Reply:
I added three branches - true/false/default